poetry/core/json/schemas/poetry-schema.json

```diff
@@ -559,6 +559,10 @@
                 "secondary": {
                     "type": "boolean",
                     "description": "Declare this repository as secondary, i.e. it will only be looked up last for packages."
+                },
+                "trusted": {
+                    "type": "boolean",
+                    "description": "Declare this repository as trusted, i.e. option --trusted-host will be used when installing packages."
```

Comment:
Sorry, I should have been a bit more clear in my last comment. Since the `--trusted-host` option is more to do with `pip`, and is more for the cli. We should be explicit about what trusted means here. I am thinking we should either call it `tls-verify` or have it as a table `tls = { verify = false }`. Latter allows for further context options. 

This also makes me wonder if this belongs in the `pyproject.toml` or in poetry config similar to auth details. The rationale being, having it in the `toml` means it could become a consideration. Usability vs security I guess. :)

Reply:
While i understand the sentiment, i believe that all those who use poetry also use `pip` separately for some cases, all of them will know the `trusted-host` option by heart, and its a better idea to keep this naming consistent, especially when this is the exact argument passed to `pip` when this is turned on.     
Still, I've added the description from the `pip` reference to make it clearer.

Reply:
Just to be clear this will no longer be passed to pip. Because pip does not talk to the indices, the packages are discovered and downloaded by poetry itself. I reckon a documentation entry hinting that this is similar to `trusted-host` in `pip` should suffice.

Reply:
Updated docs to reflect that its only similar to `pip` rather then actually passed to pip.